build/build-static-binaries.sh

```diff
@@ -12,17 +12,26 @@ source $(dirname $0)/build-common.sh
 # commands in the if-branch to be executed within the docker
 # container.
 if [ "${1-}" = "docker" ]; then
+    test_build_dir=$(mktemp -d test-binaries.XXXX)
     time make STATIC=1 build
-    time make STATIC=1 testbuild PKG=./sql
-    time make STATIC=1 testbuild PKG=./acceptance TAGS=acceptance
+    # sql/sql.test is built standalone as well to simplify the nightly logictest runs.
+    time make STATIC=1 testbuild STRIPPED=1 PKG=./sql
+    time make STATIC=1 testbuild STRIPPED=1 PKG=./acceptance TAGS=acceptance
+    time make STATIC=1 testbuild STRIPPED=1 DIR=${test_build_dir}
```

Comment:
let's keep the `Makefile` clean. There's no reason to have `DIR` leak into it with my above suggestions afaict.